### ISSUE TITLE
Revert back to features like explained in the 'cahier des charges' of Stefanie

### ISSUE THREAD

**Issue by yacineMahdid** [BIAPT/EEGsonic#36]

**Comment by yacineMahdid** [comment]
Spectral Power Ratio is now averaged throughout the channels.

**Comment by yacineMahdid** [comment]
There was a problem with fp_wpli that was falling to far behind and was getting stuck.

**Comment by yacineMahdid** [comment]
We need to ensure that either it can calculate for its alloted time (10 seconds) or that it can do it in 5 seconds which is the sampling frequency. For now I leave it at 5 seconds because I have no additional change to do and the lag created might be too big to be meaningfull.

**Comment by yacineMahdid** [comment]
There is a problem with the real-time calculation. Some features are waiting X amount of time for some data to accumulate. Then they are calculating for Y amount of time. It's very difficult to make sure the Y amount of time is always below the sampling rate as it is highly variables depending on the parameters that are choosen (i.e. the number of channels mostly).

Since we already have a lag of about X seconds I will let the analysis calculate the data in Y amount of time no matter what. I'll let the user know what is the lag for a given analysis technique when he select the parameters before the run time and then when the analysis technique is done calculating it will just try to catch up with wherever it was before.

**Comment by yacineMahdid** [comment]
The only thing we'll need to make sure is that it doesn't fall behind the number of data it is waiting for to come in. For instance Y < X.

**Comment by yacineMahdid** [comment]
Will need to remove the replay functionalities for now as it is interfering with the above stated functionality

**Comment by yacineMahdid** [comment]
I have settled in a "real-time" model that is simpler to implement and that leaves every feature as it is. 
We let the analysis technique calculate whatever it has to calculate for the time it needs. When its done it output its result and then start back its loading at the current index of the recording. This means that we wasted some data in between the calculation and the current time, however this stays in real time with a reasonable lag.
